src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -153,6 +191,15 @@
     @JsonIgnore
     private Set<ExerciseHint> exerciseHints = new HashSet<>();
 
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "EXERCISE_GROUP_ID")
+    @JsonIgnore
+    private ExerciseGroup exerciseGroup;
+
+    @ManyToMany(mappedBy = "exercises")
+    @JsonIgnore
+    private Set<StudentExam> studentExams = new HashSet<>();
```

Comment:
please leave this out for now, because we don't need this reference to be bidirectional